deployments/kustomize/README.md

```diff
@@ -57,7 +59,22 @@ resources:
 namespace: 123456-tools
 ```
 
-#### Sample Overlay for WordPress deployment to dev namespace
+### Deploying images and WordPress deployment using overlays
+* Login to OpenShift via the `oc` command line.
+* Deploy your images to OpenShift ```oc apply -k my-image-deploy``` 
+  * Deploying the images, will **not** trigger a build, this has to be done manually in OpenShift.
+
+
+## OpenShift Deployment With OpenShift Secrets
+* Create Kustomization.yaml file for example my-wordpress-deploy/kustomization.yaml
+  * Update your namespace with your license plate, and env you to deploy (dev|test|prod)
+  * Update your images to point to your tools repo with your license plate.
+* Add your patch file my-wordpress-deploy/patch.yaml
+  * include the route with the update host url
```

Comment:
Maybe this is clearer if you know more about OpenShift than I do but I'm not sure what the update host url is.